gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -385,11 +403,13 @@ class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenOrcPartitionDiscoverySuite]
     .exclude("read partitioned table - normal case")
     .exclude("read partitioned table - with nulls")
+    .excludeAll("Blocked by ORC Velox upstream not ready")
```

Comment:
Why not just comment the lines out? I think that will be less confusing to developers.

Reply:
There are lots of lines to comment out, meanwhile need to remove the "imports" to pass style check because those classes are not used after comment. We can simply remove this line after ORC is supported.

Reply:
e.g.

```scala
// Blocked by ORC Velox upstream not ready
//  enableSuite[GlutenOrcPartitionDiscoverySuite]
//    .exclude("read partitioned table - normal case")
//    .exclude("read partitioned table - with nulls")
```

Reply:
> There are lots of lines to comment out, meanwhile need to remove the "imports" to pass style check because those classes are not used after comment. We can simply remove this line after ORC is supported.

Sounds fair to me. Let me try untangle this part of code a bit directly on the PR.

Reply:
https://github.com/oap-project/gluten/pull/3402